src/main/java/org/folio/inventory/resources/UpdateOwnershipApi.java

```diff
@@ -291,6 +295,39 @@ private CompletableFuture<List<String>> updateOwnershipOfHoldingsRecords(Holding
     }
   }
 
+  private CompletableFuture<List<UpdateOwnershipHoldingsRecordWrapper>> validateHoldingsRecordBoundWithParts(List<UpdateOwnershipHoldingsRecordWrapper> holdingsRecordWrappers,
```

Comment:
please add inventory-storage.bound-with-parts.collection.get permission for the /holdings/update-ownership endpoint

Reply:
Done, thank you!

Reply:
thank you!